app/controllers/homepage_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomepageController < ApplicationController
```

Comment:
I'm not sure why the comment is still here. Below it says " No violations found. Woof!". I think this won't show up in future ones now that the hound config has been merged.